src/inet.c

```diff
@@ -509,9 +512,10 @@ static conn_t *init_conn(pool *p, int fd, const pr_netaddr_t *bind_addr,
         pr_signals_unblock();
       }
 
-      if (reporting) {
-        pr_log_pri(PR_LOG_ERR, "Failed binding to %s, port %d: %s",
-          pr_netaddr_get_ipstr(&na), port, strerror(hold_errno));
+      pr_log_pri(PR_LOG_ERR, "Failed binding to %s, port %d: %s",
+        pr_netaddr_get_ipstr(&na), port, strerror(hold_errno));
```

Comment:
This message is now logged all of the time (as was happening most of the time previously); the messages about checking `ServerType` are only relevant for the daemon listening sockets, not for session-specific data transfer sockets.